src/idom_bokeh/src/index.ts

```diff
@@ -1,11 +1,37 @@
 import { mountLayout } from "idom-client-react";
 
+import {ModelEvent, JSON} from "@bokehjs/core/bokeh_events"
 import * as p from "@bokehjs/core/properties";
 import { HTMLBox } from "@bokehjs/models/layouts/html_box";
 import { register_models } from "@bokehjs/base";
 
 import { PanelHTMLBoxView, set_size } from "./layout";
 
+export class IDOMEvent extends ModelEvent {
+  event_name: string = "idom_event"
+
+  constructor(readonly data: any) {
+    super()
+  }
+
+  protected _to_json(): JSON {
+    const events = []
+    console.log(this.data)
+    for (let e of this.data.data) {
+      e = {...e}
+      if (e.target?.boundingClientRect != null) {
+	e.target.boundingClientRect = {...e.target.boundingClientRect}
+      }
+      if (e.currenttarget?.boundingClientRect != null) {
+	e.target.boundingClientRect = {...e.currentTarget.boundingClientRect}
+      }
```

Comment:
Can you explain a bit of the issue here? I haven't encountered this when serializing with `JSON.stringify`. Does this need a fix in IDOM core's event-to-object implementation? It should be simple enough to use a spread operator [here](https://github.com/idom-team/idom/blob/main/src/client/packages/idom-client-react/src/event-to-object.js#L55).

Reply:
I think Bokeh just doesn't serialize custom objects so the spread operator to turn it into a regular object is indeed all that's needed.